client/verta/tests/test_cli/test_endpoint.py

```diff
@@ -409,6 +410,45 @@ def test_update_autoscaling(self, client, created_endpoints, experiment_run, mod
                 assert metric["parameters"][0]["name"] == "target"
                 assert metric["parameters"][0]["value"] == "0.7"
 
+class TestDownload:
+    def test_download_context(self, experiment_run, model_for_deployment, registered_model, in_tempdir, created_registered_models):
+        np = pytest.importorskip("numpy")
+        version_name = "my-version"
+        experiment_run.log_model(model_for_deployment['model'], custom_modules=[])
+        experiment_run.log_requirements(['scikit-learn'])
+
+        artifact = np.random.random((36, 12))
+        experiment_run.log_artifact("some-artifact", artifact)
+        model_version = registered_model.create_version_from_run(experiment_run.id, version_name)
+
+        download_to_path = "context_cli.tgz"
+
+        runner = CliRunner()
+        result = runner.invoke(
+            cli,
+            ['deployment', 'download', 'dockercontext', '--run_id', experiment_run.id, '--output', download_to_path],
+        )
+        assert not result.exception
+
+        # can be loaded as tgz
+        with tarfile.open(download_to_path, 'r:gz') as f:
+            filepaths = set(f.getnames())
+
+        assert "Dockerfile" in filepaths
+
+        runner = CliRunner()
+        result = runner.invoke(
+            cli,
+            ['deployment', 'download', 'dockercontext', '--model_version_id', model_version.id, '--output', download_to_path],
```

Comment:
For the model version test, I think we should use a new ModelVersion _not_ `from_run()`.
So, using `model_version.log_model()`, `log_artifact()`, and `log_environment()`

Reply:
replaced